src/main/java/org/cascadebot/cascadebot/commands/informational/ServerInfoCommand.java

```diff
@@ -23,9 +23,18 @@
     @Override
     public void onCommand(Member sender, CommandContext context) {
         Guild guildForInfo = context.getGuild();
+        if (context.getArgs().length != 0 && !context.isArgInteger(0)) {
```

Comment:
This will make ;serverinfo on it's own fail. Since you've added a check below, you can just remove this

Reply:
guildinfo on it's own does not fail, but you're right its not needed 👍